includes.h

```diff
@@ -16,14 +16,41 @@
 #ifndef INCLUDES_H
 #define INCLUDES_H
 
+#ifdef __TANDEM
+/*
+ * This must be done here instead of on the configure command line to
+ * avoid missing the socklen_t declaration in header files.
+ */
+#define _XOPEN_SOURCE_EXTENDED 1
```

Comment:
Can you do this in configure.ac with a -D flag?  That's how it's done for, eg, hpux:
CPPFLAGS="$CPPFLAGS -D_HPUX_SOURCE -D_XOPEN_SOURCE -D_XOPEN_SOURCE_EXTENDED=1"

Reply:
I would love to oblige. The comment above this line explains that the openssh-portable configure script misidentifies socklen_t and fails to complete. I can post the config.log file that shows this. It seems like a header conflict, which is annoying. I forgot to wrap this in an `#ifdef`, though. Will fix this ASAP.

Reply:
if it's a single test you might be able to copy the socklen_t test into the configure block for Tandem immediately before this.  configure will then cache the result.

Reply:
I have to admit that I do not really know how to pull that off but it is a better solution, I agree

Reply:
If you can post the configure output and config.log someplace I can review and see if I can figure out what's going on.  (If you don't have any better place, maybe also open a bug at bugzilla.mindrot.org and point to this PR).

Reply:
It looks like putting `-D_XOPEN_SOURCE_EXTENDED=1` into `configure.ac` gets past this one. I will commit `configure.ac` separately.